modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides the support for legacy BIOS backward compatibility with UEFI systems.
+
+[NOTE]
+====
+When you are deploying with iRMC, you must configure the CSM, otherwise the cluster installation might fail.
+====
+
+[IMPORTANT]
+====
+Before you configure the CSM, ensure that you have disabled the `Secure Boot Control`, which is available under `Security` -> `Secure Boot Configuration`.
+====
+
```

Comment:
You could include a `.Procedure` heading here. For consistency, you could do the same in the other procedures in this module directly prior to where the procedural steps begin.

Reply:
I have put the `.Procedure` prior to where the procedural steps begin. PTAL.

Reply:
Thank you.